test/common.c

```diff
@@ -408,3 +408,26 @@ void replace_strdup(gchar **dst, const gchar *src)
 	g_free(*dst);
 	*dst = g_strdup(src);
 }
+
+void* dup_test_mem(GPtrArray *ptrs, const void *mem, gsize len)
```

Comment:
Maybe it should somehow reflect that something is added to the list of 'to free' candidates? But I also don't really have a good name that covers both the copying and the freeing aspect...

Reply:
Yes, I've thought about that for some time as well and this is the best I came up with. Can we keep it for now?